_posts/2018/2018-04-01-stop-robots-crawlers-triggering-errors-rails.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 'Stop robots and crawlers causing errors in your Rails application'
+description: 'Beware the remorseless robot army.'
+layout: article
+category: ruby
+image:
+  base: '2018/stop-robots-crawlers-triggering-errors-rails'
+  alt: 'BB-8'
+  source: 'https://unsplash.com/photos/C8VWyZhcIIU'
+  credit: 'Joseph Chan'
+---
+
+When a Rails application can't find a record, it throws a 404 error. This is a standard HTTP code for browsers (and other clients) meaning 'not found'.
+
+When you have an Internet-facing site various search engines will be crawling your site. As you change things certain URLs might no longer work and the search engines can start generating a lot of ‘not found’ errors in your application by hitting all the links that used to exist.
```

Comment:
*...various search engines will be crawling _it_. 

Then have a one-liner about what crawling is.

Reply:
Add a comma after "As you change things"

*As you change things, certain URLs might no longer work and the search engines/crawlers(?) can start generating a lot of 'not found' errors in your application when it hits them.